src/language/sv.json

```diff
@@ -54,6 +54,8 @@
     "new-yritys": "Nytt företag?",
     "yritys": "Företag",
     "yritys-laatijat": "Upprättare",
+    "ohjeet": "Laatijan ohjeet SV?",
```

Comment:
Nyt kun meillä on tuo scripti mikä generoi tämän sv.jsonin, niin parempi kirjoittaa nämä vain fi.jsoniin ja ajaa sen scriptin jälkeenpäin. Ei tule mahdollisuuksia typoihin jne. Toki tuo tuskin oli käytössä tätä tehdessä, mutta jatkoa varten muistuttelen.